docs/content/deployment/run-retries.mdx

```diff
@@ -0,0 +1,51 @@
+---
+title: Run retries | Dagster
+description: Automatically retry Dagster runs
+---
+
+# Run Retries
+
+If you configure retries at the Job level, failed runs will trigger retries in a new run. Compared
+to [Op retries](/concepts/ops-jobs-graphs/op-retries#op-retries), the max retry limit for run retries
+applies to the whole run instead of each individual Op. Run retries also handle the cases where a run worker crashed.
+
+## Configuration
+
+To enable run retries, add the following to your `dagster.yaml`. This will start a new daemon which polls
+to the event log for run failure events.
+
+```yaml file=/deploying/dagster_instance/dagster.yaml startafter=start_run_retries endbefore=end_run_retries
+# Opt in to run retries
+run_retries:
+  enabled: true
+  max_retries: 3 # Sets a default for all jobs. 0 if not set
+```
+
+You can also configure retries using tags on Job definitions or in Dagit.
```

Comment:
could it be worth including a dagit screenshot here to show how to do that in dagit?

Reply:
How to edit tags in the launchpad? Maybe on the Dagit page

Reply:
Changed wording to make it more clear that it would be the launchpad tags

Reply:
yea guess the minimum we could link to https://docs.dagster.io/concepts/dagit/dagit#launchpad